engine/api/src/main/scala/pl/touk/nussknacker/engine/api/definition/ParameterValidator.scala

```diff
@@ -16,12 +19,34 @@ import pl.touk.nussknacker.engine.api.context.ProcessCompilationError.{EmptyMand
  */
 @ConfiguredJsonCodec sealed trait ParameterValidator {
 
-  def isValid(paramName: String, expression: String)(implicit nodeId: NodeId): Validated[PartSubGraphCompilationError, Unit]
+  def isValid(paramName: String, value: String, label: Option[String] = None)(implicit nodeId: NodeId): Validated[PartSubGraphCompilationError, Unit]
```

Comment:
Default value is ugly.. Better way is create two methods with different signature: one with label second without.

Reply:
Removed default value